en/FindDuplicates.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Find duplicates
+---
+
+# Find duplicates
+
+JabRef can look for duplicated entries inside a database.
+
+This feature is accessible directly through  **Quality -&gt; Find duplicates**.
+It is also used when [importing new entries](ImportInspectionDialog)
+from a supported reference format or directly from the Internet.
+
+Detection of potential duplicates is done by a distance algorithm.
```

Comment:
Included (with direct commit, since I did not know how to deal with your comments in this already-merged PR).